pandas/tests/apply/test_frame_apply.py

```diff
@@ -995,7 +995,7 @@ def test_consistency_for_boxed(box, int_frame_const_col):
 
 
 def test_agg_transform(axis, float_frame):
-    other_axis = 1 if axis in {0, "index"} else 0
+    other_axis = 1 if DataFrame()._get_axis_number(axis)==0 else 0
```

Comment:
the issue has been revised to avoid fixing `test` files. Too many new contributors were making mistakes by changing the nature of what was being tested.

Reply:
So, no touching test files, then?

Reply:
dont edit test files.

Reply:
Aight. Won't. Thanks for telling. I'll revert the changes